IM_calculation/scripts/calculate_ims.py

```diff
@@ -29,7 +29,7 @@ def load_args():
 
     parser = argparse.ArgumentParser(parents=[parent_parser], add_help=True)
     #
-    #parser = argparse.ArgumentParser()
```

Comment:
Remove these lines.

Reply:
there is one more test that fails, i'll remove all those comments after the test are all passed ( keeping them for reference just for now)

Reply:
they should be all removed now.